Koios Java Client Bump to 1.16.2 + CIP30 Value Serialize/Deserialize

edridudi opened the pull request.

satran004 commented:
@edridudi Please check the build error

satran004 commented on cip/cip30/src/main/java/com/bloxbean/cardano/client/cip/cip30/CIP30ValueDeserializer.java:
```diff
@@ -0,0 +1,48 @@
+     * @param bytes balance cbor bytes from wallet.getBalance()
+     * @return {@link Value}
+     */
+    public static Value deserialize(byte[] bytes) throws CborDeserializationException {
```
@edridudi If we move Value's deserialization code from TransactionOutput to Value class, do we still need this class? I understand why you have introduced this class now, but it may make sense to keep this logic in one place for easy maintenance in future. Because now there are 3 different ways value can be represented between Shelley to Babbage. For any change in future, it has to be done in both places.

satran004 commented on cip/cip30/src/main/java/com/bloxbean/cardano/client/cip/cip30/CIP30ValueSerializer.java:
```diff
@@ -0,0 +1,50 @@
+ * Serialize Value Object to cbor CIP30's Balance Object
+ * to Support CIP30's wallet.getUtxo(amount: cbor&lt;value&gt;) method
+ */
+public class CIP30ValueSerializer {
```
Same as CIP30Deserializer, do we need this class ? We can keep this class but it will invoke Value class.

edridudi replied:
We don't need these Classes as long as the serialization & deserialization functions have the same functionality as in these classes.
I agree that we should have them in one single place. that would be even better.

edridudi commented:
@satran004 ,
Updated PR with CIP30ValueDeserializer.java, CIP30ValueSerializer.java, CIP30ValueTest.java Removed.
Moved CIP30ValueTest Unit-Tests to ValueSpecTest (Errors in this PR will be resolved after merging with #210)

satran004 commented:
@edridudi PR #210 is merged now. Can you please update your repo and remove CIP30ValueDeserializer.java ? Thanks
